libcaf_core/test/mixin/sender.cpp

```diff
@@ -55,28 +55,50 @@ struct fixture : test_coordinator_fixture<> {
 
 CAF_TEST_FIXTURE_SCOPE(sender_tests, fixture)
 
-CAF_TEST(delayed actor message) {
+CAF_TEST(delayed actor messages receive responses) {
   self->delayed_send(testee, seconds(1), "hello world");
   sched.trigger_timeout();
   expect((std::string), from(self).to(testee).with("hello world"));
```

Comment:
Maybe use a variable in the fixture for the teststring to not write it out every time.
```suggestion
  expect((std::string), from(self).to(testee).with(test_string));
```